test/extensions/filters/http/grpc_json_transcoder/grpc_json_transcoder_integration_test.cc

```diff
@@ -218,18 +218,47 @@ TEST_P(GrpcJsonTranscoderIntegrationTest, UnaryPost) {
       R"({"id":"20","theme":"Children"})");
 }
 
+TEST_P(GrpcJsonTranscoderIntegrationTest, TestParamUnescapePlus) {
```

Comment:
Is there also a unit test?

Reply:
unit-test is done in grpc-httpjson-transcoding [repo](https://github.com/grpc-ecosystem/grpc-httpjson-transcoding/pull/60).